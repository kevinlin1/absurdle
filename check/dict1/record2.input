adieu
words
blank
fancy
jazzy
happy
mammy
tatty
