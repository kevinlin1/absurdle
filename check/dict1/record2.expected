2309 words
> adieu
: 🟨⬜⬜⬜⬜

284 words
> words
: ⬜⬜⬜⬜⬜

64 words
> blank
: ⬜⬜🟨⬜⬜

16 words
> fancy
: ⬜🟩⬜⬜🟩

5 words
> jazzy
: ⬜🟩⬜⬜🟩

4 words
> happy
: ⬜🟩⬜⬜🟩

2 words
> mammy
: ⬜🟩⬜⬜🟩

1 words
> tatty
: 🟩🟩🟩🟩🟩

Absurdle 8/∞

🟨⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜⬜🟨⬜⬜
⬜🟩⬜⬜🟩
⬜🟩⬜⬜🟩
⬜🟩⬜⬜🟩
⬜🟩⬜⬜🟩
🟩🟩🟩🟩🟩
